ribs/schedulers/_scheduler.py

```diff
@@ -209,6 +229,11 @@ def _tell_internal(self,
             status_batch = np.asarray(status_batch)
             value_batch = np.asarray(value_batch)
 
+        # Add solutions to result_archive.
+        if self._result_archive is not None:
+            self._result_archive.add(self._solution_batch, objective_batch,
```

Comment:
We should add_single in single mode (also mention this in the documentation for the add_mode).

Reply:
This way, the archives have as close behavior as possible.